pkg/storage/sst.go

```diff
@@ -54,6 +54,27 @@ func CheckSSTConflicts(
 	maxIntents int64,
 	usePrefixSeek bool,
 ) (enginepb.MVCCStats, error) {
+
+	// Check for any range keys.
+	//
+	// TODO(erikgrinaker): We should support these, but it's not needed yet.
```

Comment:
I think we need a GH issue for reference here and below as well.

Reply:
Yeah, we have #76234, linked it from the `TODO` comments.